watt.py

```diff
@@ -172,6 +173,27 @@ def download_gcs_blob_to_tmp_file(self, blob_path: str, tmp_file_name: str) -> N
         blob = bucket.blob(blob_str)
         blob.download_to_filename(tmp_file_name)
 
+    @staticmethod
+    def read_in_chunks(file_object, chunk_size=1024*1024):
```

Comment:
Does this mean if a line in a file is over 1024*1024 characters this could fail? Or would it start reading from the next portion in the following iteration of `readlines`?

Reply:
no, the parameter in `readlines` tells `readlines` not to read any more lines if the current total is above the specified size.  But it will read more than the specified size in order to ingest a line if it hasn't yet passed the chunk_size.

https://docs.python.org/3/library/io.html#io.IOBase.readlines